### DISCUSSION TITLE
AXIS-P8221 I/O module  Scripting

### DISCUSSION THREAD

**Discussion by Dallasminer** [AxisCommunications/discussions#56] [ACAP]
situtaion:
using i/0 module to trigger the recorder elk 120 board but having issue  of the i/o not turning off after the first trigger .

is there a way to add a script on axis i/o module turn itself off after the first input signal or trigger ?

**Comment by mattias-kindborg-at-work** [comment]
Hi @Dallasminer! Let me see if I can find an answer.

**Reply by Dallasminer** [reply]
@mattias-kindborg-at-work Thank you so much i really appreciate that

**Reply by Dallasminer** [reply]
@mattias-kindborg-at-work  any update sir

**Reply by mattias-kindborg-at-work** [reply]
Would you like the I/O to remain open for a period of time or instantly switch back?
